cpu/mips_pic32mx/ldscripts/pic32mx512_12_128_uhi.ld

```diff
@@ -0,0 +1,324 @@
+/*
+ * For all MX devices with 512K program flash / 12KB boot flash and 128KB Ram
+ *
+ * A platform and target independent link script to produce UHI
+ * compliant binaries with varying levels of system initialization
+ * support.
+ */
+
```

Comment:
I think maybe this should be located in the mips_common folder, and add a linker script for each supported cpu with only the memory map (as done for cortex platforms). It will look cleaner and flexible when other cpu's will be added.